Formula/digdag.rb

```diff
@@ -0,0 +1,24 @@
+class Digdag < Formula
+  desc "Workload Automation System"
+  homepage "https://github.com/treasure-data/digdag"
+  url "https://dl.digdag.io/digdag-0.8.1.jar"
+  sha256 "92e79f5eb26b8992c9d510206c71f64fe4765c0c51a4e4cb62279aa1a4231ec7"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    # Add prefix a shebang before install for OS X 10.9.
+    # See https://github.com/Homebrew/homebrew-core/pull/2012#issuecomment-226407948 for the detail.
+    inreplace "digdag-#{version}.jar", /\A/, "#!/bin/sh\n"
```

Comment:
It does work when started from a shell. It doesn't work when started directly from Ruby via `system "#{bin}/digdag"`. This possibly only affects old Rubies as it was fine on 10.10/10.11 and just failed on our 10.9 test bot. Still, having a shebang line is desirable in my opinion. (I'm just not sure we can sell adding one to upstream for the reasons explained above.)